RELEASE.md

```diff
@@ -37,13 +37,13 @@ An older Python version may be removed from the automatic release cycle if:
 
 For the actively supported versions as of December 2023, the following release cycle is expected:
 
-| Python version | Scheduled/expected end of active support |
-| -------------- | ---------------------------------------- |
-| Python 3.8     | January 2024                             |
-| Python 3.9     | July 2024                                |
-| Python 3.10    | January 2025                             |
-| Python 3.11    | January 2026                             |
-| Python 3.12    | January 2027                             |
+| Python version | Currently supported | Scheduled/expected end of active support |
+| -------------- | ------------------- | ---------------------------------------- |
+| Python 3.8     | :x:                 | 10-January-2024 (dropped)                |
+| Python 3.9     | :heavy_check_mark:  | July 2024 (expected)                     |
+| Python 3.10    | :heavy_check_mark:  | January 2025 (expected)                  |
+| Python 3.11    | :heavy_check_mark:  | January 2026 (expected)                  |
+| Python 3.12    | :heavy_check_mark:  | January 2027 (expected)                  |
```

Comment:
please use the unicode symbols in https://github.com/PowerGridModel/.github/blob/main/SECURITY.md?plain=1#L16-L17